app/views/landmarks/show.html.erb

```diff
@@ -1,16 +1,75 @@
-<h1 class='landmark-name'><%= @landmark.name %></h1>
-<h3 class='category'><%= @landmark.category.capitalize %></h3>
-<ul class='attributes'>
-  <li>Phone: <%= @landmark.phone_number %></li>
-  <li>Website: <%= @landmark.address %></li>
-</ul>
-
-<br>
-<% if current_user %>
-  <%= button_to "Upload new recording", new_landmark_recording_path(@landmark), method: :get, class: 'btn btn-primary' %>
-<% end %>
-<br>
-<br>
-<% @landmark.recordings.each do |recording| %>
-  <%= render partial: 'shared/audio_player', object: recording %>
+<% provide :head_tags do %>
+  <meta name='turbolinks-visit-control' content='reload'>
+  <script>
+    document.addEventListener("DOMContentLoaded", function(){
+      initMap(<%=facade.landmark.lat%>, <%=facade.landmark.long%>)
+    });
+  </script>
 <% end %>
+
+<!DOCTYPE html>
```

Comment:
What's going on here? None of the stuff below should be in the view itself. Maybe part of it needs to go in the head in `application.html.erb`.

Reply:
Sorry, I forgot to get rid of this. I'll clean it up.